src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json

```diff
@@ -66,9 +122,97 @@
                 "type": "string",
                 "metadata": {
                     "displayName": "Denied groups for SSH",
-                    "description": "List of user groups to be denied to connect with SSH. Default is root"
+                    "description": "List of user groups to be denied to connect with SSH. Default is 'root'"
                 },
                 "defaultValue": "root"
+            },
+            "HostBasedAuthentication": {
+                "type": "string",
+                "metadata": {
+                    "displayName": "Host-based authentication",
+                    "description": "Host-based authentication. Default is  'no'"
+                },
+                "defaultValue": "no"
+            },
+            "PermitRootLogin": {
+                "type": "string",
+                "metadata": {
+                    "displayName": "Whether root can log in using ssh",
+                    "description": "Whether root can log in using ssh. Default is 'no'"
+                },
+                "defaultValue": "no"
+            },
+            "PermitEmptyPasswords": {
+                "type": "string",
+                "metadata": {
+                    "displayName": "Whether the server allows login to accounts with empty password strings",
+                    "description": "Whether the server allows login to accounts with empty password strings. Default is  'no'"
```

Comment:
Double space here too between `is` and `'no'`

Reply:
Another above on line 133 for `HostBasedAuthentication`

Reply:
All of them had this. All are fixed now.